OMCompiler/SimulationRuntime/c/simulation/solver/initialization/initialization.c

```diff
@@ -248,7 +256,7 @@ static int symbolic_initialization(DATA *data, threadData_t *threadData)
   MMC_CATCH_INTERNAL(simulationJumpBuffer)
 #endif
     if (adaptiveGlobal)
-      data->callback->useHomotopy = 2;
+      data->callback->useHomotopy = 2; /* new global homotopy approach (adaptive lambda) */
```

Comment:
These kind of comments are very useful, thanks! 👍 It might even be worth defining an enumeration for it.